docs/index.md

```diff
@@ -138,6 +139,16 @@ Note that to see the verbose logs you need to pass in `-log=verbose` to clangd.
 You can find details about accessing clangd logs in
 https://clangd.llvm.org/troubleshooting.html#gathering-logs.
 
+### Unknown key config warning
```

Comment:
`Unknown config key warning`

Reply:
done in 6f96b5b9f36de016d9bcf6feaa4443374d27ca1f